src/Config.php

```diff
@@ -1597,6 +1597,9 @@ public static function checkExtensions($list = null)
                 'simplexml' => [
                     'required'  => true,
                 ],
+                'bcmath' => [
```

Comment:
If the extension is required, please add it to the composer requirements too (`composer require ext-bcmath`).

Reply:
@cedric-anne 
this https://github.com/glpi-project/docker-images/pull/131 is still needed right?

Reply:
yes